src/pages/WishListPage/WishListPage.jsx

```diff
@@ -1,16 +1,25 @@
 import React from "react";
-import { Productcard, WishListCard } from "../../components/index";
-// import "../../components/ProductCard/Productcard.css";
+import {
+  ErrorMessage,
+  Productcard,
+  WishListCard,
+} from "../../components/index";
+import { useCartandWishList } from "../../context/CartAndWishlist-context";
 
 export const WishListPage = () => {
+  const { wishListState, wishListDispatch } = useCartandWishList();
+  const { wishListItem, wishListCount } = wishListState;
   return (
     <div className="main-content-container">
       <h2 className="heading">My WishList</h2>
       <div className="grid-3-column-layout">
-        <WishListCard />
-        <WishListCard />
-        <WishListCard />
-        <WishListCard />
+        {wishListItem.length === 0 ? (
+          <ErrorMessage message={"No Products in your WishList Page"} />
+        ) : (
```

Comment:
You can add a to go to productListing page if no item present in wishlist

Reply:
okay will do